### PR TITLE
docs: add event api guide

### PR DESCRIPTION
**Pull request by hifabienne**
- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/guides/integrate/event-api.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+title: Get events from ZITADEL
+---
+
+ZITADEL leverages the power of eventsourcing, meaning every action and change within the system generates a corresponding event that is stored in the database. 
+To provide you with greater flexibility and access to these events, ZITADEL has introduced an Event API. 
+This API allows you to easily retrieve and utilize the events generated within the system, enabling you to integrate them into your own system and respond to specific events as they occur.
+
```
Comment (mffap) [inline docs/docs/guides/integrate/event-api.md:8]:
```suggestion
This API allows you to easily retrieve and utilize the events generated within the system, enabling you to integrate them into your own system and respond to specific events as they occur.

You need to give a user the [manager role](https://zitadel.com/docs/guides/manage/console/managers) IAM_OWNER_VIEWER or IAM_OWNER to access the Event API.

```

File: docs/sidebars.js [hunk 2]
```diff
@@ -140,6 +140,7 @@ module.exports = {
           ],
         },
         "guides/integrate/logout",
+        "guides/integrate/event-api",
```
Comment (mffap) [inline docs/sidebars.js:143]: I would suggest that we put it in Integrate>"Access ZITADEL APIs"

File: docs/docs/guides/integrate/event-api.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+- event types
+- aggregate id
+- aggregate types
+- resource owner
+- creation date
+
+```bash
+curl --request POST \
+  --url $YOUR-DOMAIN/admin/v1/events/_search \
+  --header 'Authorization: Bearer $TOKEN'
```
Comment (mffap) [inline docs/docs/guides/integrate/event-api.md:25]:
```suggestion
  --header "Authorization: Bearer $TOKEN"
```

File: docs/docs/guides/integrate/event-api.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+```
+
+## Get event types
+
+To be able to filter for the different event types ZITADEL knows, you can request the [EventTypesList](../../apis/proto/admin#listeventtypes)
+
+```bash
+curl --request POST \
+--url $YOUR-DOMAIN/admin/v1/events/types/_search \
+--header 'Authorization: Bearer $TOKEN' \
```
Comment (mffap) [inline docs/docs/guides/integrate/event-api.md:35]:
```suggestion
--header "Authorization: Bearer $TOKEN" \
```

File: docs/docs/guides/integrate/event-api.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+```
+
+## Get aggregate types
+
+To be able to filter for the different aggregate types (resources) ZITADEL knows, you can request the [AggregateTypesList](../../apis/proto/admin#listaggregatetypes)
+
+```bash
+curl --request POST \
+  --url $YOUR-DOMAIN/admin/v1/aggregates/types/_search \
+  --header 'Authorization: Bearer $TOKEN' \
```
Comment (mffap) [inline docs/docs/guides/integrate/event-api.md:71]:
```suggestion
  --header "Authorization: Bearer $TOKEN" \
```

File: docs/docs/guides/integrate/event-api.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+Assuming you use ZITADEL as your single source of truth for your user data.
+Now you like to react to changes on the users to update data in other your other systems.
+
+This example shows you how to get all events from users, filtered with the creation_date (e.g since last day/hour, etc).
+
+```bash
+curl --request POST \
+  --url $YOUR-DOMAIN/admin/v1/events/_search \
+  --header 'Authorization: Bearer $TOKEN' \
```
Comment (mffap) [inline docs/docs/guides/integrate/event-api.md:102]:
```suggestion
  --header "Authorization: Bearer $TOKEN" \
```

File: docs/docs/guides/integrate/event-api.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+
+## Example: Find out when user have been authenticated
+
+The following example shows you how you could use the events search to get all events where a token has been created.
+Also we include the refresh tokens in this example to know when the user has become a new token.
+
+```bash
+curl --request POST \
+  --url $YOUR-DOMAIN/admin/v1/events/_search \
+  --header 'Authorization: Bearer $TOKEN' \
```
Comment (mffap) [inline docs/docs/guides/integrate/event-api.md:122]:
```suggestion
  --header "Authorization: Bearer $TOKEN" \
```

File: docs/docs/guides/integrate/event-api.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+In this case this are the following events:
+- Password verification failed
+- One-time-password (OTP) check failed (Authenticator Apps like Authy, Google Authenticator, etc)
+- Universal-Second-Factor (U2F) check failed (FaceID, WindowsHello, FingerPrint, etc)
+- Passwordless/Passkey check failed (FaceID, WindowsHello, FingerPrint, etc)
+
+```bash
+curl --request POST \
+  --url $YOUR-DOMAIN/admin/v1/events/_search \
+  --header 'Authorization: Bearer $TOKEN' \
```
Comment (mffap) [inline docs/docs/guides/integrate/event-api.md:148]:
```suggestion
  --header "Authorization: Bearer $TOKEN" \
```

**Review by mffap — CHANGES_REQUESTED** [review]

File: docs/sidebars.js (hunk 2 above)
Comment (hifabienne) [reply docs/sidebars.js]: I changed it

**Review by mffap — APPROVED** [review]

**Pull request merged**